fearless/Modules/AccountCreate/AccountCreatePresenter.swift

```diff
@@ -128,10 +135,16 @@ final class AccountCreatePresenter {
 extension AccountCreatePresenter: AccountCreatePresenterProtocol {
     func setup() {
         interactor.setup()
-
+        switch flow {
+        case .wallet:
+            view?.set(chainType: .both)
+        case let .chain(model):
+            selectedCryptoType = CryptoType(rawValue: UInt8(model.meta.substrateCryptoType)) ?? .sr25519
```

Comment:
how many "?? .sr25519" we have? maybe it makes sense to create CryptoType.DefaultSubstrate constants?

Reply:
5. May be